ACPI 6.0 specified MADT generic distributor version values, but
the detail definition is missing, add its support in this patch.

Signed-off-by: Hanjun Guo <hanjun.guo@linaro.org>
---
Hi Bob, Lv,

As discussed with Lv on linux-acpi maillist, I prepared this
patch for review.

Please treat this patch as a fix for next ACPICA version and
linux kernel 4.2 material, because we still can not
use the newest ACPICA for linux when this enum is missing,
any comments are welcomed.

Thanks
Hanjun

 source/include/actbl1.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)

Index: acpica-unix-20150515/source/include/actbl1.h
===================================================================
--- acpica-unix-20150515.orig/source/include/actbl1.h
+++ acpica-unix-20150515/source/include/actbl1.h
@@ -978,6 +978,18 @@ typedef struct acpi_madt_generic_distrib
 
 } ACPI_MADT_GENERIC_DISTRIBUTOR;
 
+/* Values for Version in Generic Distributor  (ACPI 6.0) */
+
+enum AcpiMadtGicVersionType
+{
+    ACPI_MADT_GIC_VER_UNKNOWN    = 0,
+    ACPI_MADT_GIC_VER_V1         = 1,
+    ACPI_MADT_GIC_VER_V2         = 2,
+    ACPI_MADT_GIC_VER_V3         = 3,
+    ACPI_MADT_GIC_VER_V4         = 4,
+    ACPI_MADT_GIC_VER_RESERVED   = 5    /* 5 and greater are reserved */
+};
+
 
 /* 13: Generic MSI Frame (ACPI 5.1) */
 
